wandb/deploys/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-bullseye
+
+RUN apt-get update && apt-get upgrade && apt-get install
+# install git
+RUN apt-get install -y git
+RUN rm -rf /var/lib/apt/lists
+
+# required pip packages
+RUN pip install git+https://github.com/wandb/client.git@launch/general-launch-agent
+RUN pip install wandb[launch]
+RUN pip install kubernetes
```

Comment:
These can also be combined:

```dockerfile
RUN pip install --no-cache-dir git+https://github.com/wandb/client.git@launch/general-launch-agent#egg=wandb[launch] kubernetes
```

The `--no-cache-dir` tells pip not to store the downloaded wheels in the pip cache which will reduce the number of file system changes and the overall size of the image.

Reply:
Awesome, doing that now